index.html

```diff
@@ -38,5 +38,161 @@ <h2>
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="about-me">
+              <h1>About me</h1>
+              <p>
+                  Hello I’m a software developer! I can help you build a product , 
+                  feature or website Look through some of my work and experience! 
+                  If you like what you see and have a project you need coded, 
+                  don’t hestiate to contact me.
+              </p>
+              <button>Get My Resume</button>
+              <hr>
+            </div>
+            <div class="frames">
+              <div class="frame">
+                <div class="frame1">
+                  <img src="images/Rectangle 55.png" alt="">
+                  <h1>Languages</h1>
+                </div>
+                <div class="languages">
```

Comment:
Your about-me section looks closely like the design. Please, add the background image as well.

You design:
![image](https://user-images.githubusercontent.com/20554583/199916670-00f59240-9106-4c25-b05c-0ebc0dcacd10.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/199916693-c4223b8c-6d23-4f43-a7cb-24e016e91896.png)